### PR TITLE
replaced nltk_tokenize_words with WordTokenizer

### PR DESCRIPTION
**Pull request by glaserti**
as in  http://docs.cltk.org/en/latest/greek.html#word-tokenization

### CODE REVIEW

File: 3 Basic NLP.ipynb [hunk 1]
```diff
@@ -282,7 +294,8 @@
    "source": [
     "# Import general-use word tokenizer\n",
     "\n",
-    "from cltk.tokenize.word import nltk_tokenize_words"
+    "from cltk.tokenize.word import WordTokenizer\n",
```
Comment (clemsciences) [inline 3 Basic NLP.ipynb]: Good idea to update the tutorials!

File: 3 Basic NLP.ipynb [hunk 2]
```diff
@@ -53,7 +53,19 @@
    "cell_type": "code",
    "execution_count": 3,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```
Comment (clemsciences) [inline 3 Basic NLP.ipynb:56]: I think it is not a good idea to have an output which shows that cltk is not found in your Python environment!

File: 3 Basic NLP.ipynb (hunk 1 above)
Comment (clemsciences) [inline 3 Basic NLP.ipynb:297]: Good idea to update the tutorials!

File: 3 Basic NLP.ipynb [hunk 3]
```diff
@@ -1047,7 +1061,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "['αὐτὸς', 'αὐτός', 'γε', 'γὰρ', 'γάρ', \"δ'\", 'δαὶ', 'δαὶς', 'δαί', 'δαίς', 'διὰ', 'διά', 'δὲ', 'δέ', 'δὴ', 'δή', 'εἰ', 'εἰμὶ', 'εἰμί', 'εἰς', 'εἴμι', 'κατὰ', 'κατά', 'καὶ', 'καί', 'μετὰ', 'μετά', 'μὲν', 'μέν', 'μὴ', 'μή', 'οἱ', 'οὐ', 'οὐδεὶς', 'οὐδείς', 'οὐδὲ', 'οὐδέ', 'οὐκ', 'οὔτε', 'οὕτως', 'οὖν', 'οὗτος', 'παρὰ', 'παρά', 'περὶ', 'περί', 'πρὸς', 'πρός', 'σὸς', 'σός', 'σὺ', 'σὺν', 'σύ', 'σύν', 'τε', 'τι', 'τις', 'τοιοῦτος', 'τοὶ', 'τοί', 'τοὺς', 'τούς', 'τοῦ', 'τὰ', 'τά', 'τὴν', 'τήν', 'τὶ', 'τὶς', 'τί', 'τίς', 'τὸ', 'τὸν', 'τό', 'τόν', 'τῆς', 'τῇ', 'τῶν', 'τῷ', \"ἀλλ'\", 'ἀλλὰ', 'ἀλλά', 'ἀπὸ', 'ἀπό', 'ἂν', 'ἄλλος', 'ἄν', 'ἄρα', 'ἐγὼ', 'ἐγώ', 'ἐκ', 'ἐξ', 'ἐμὸς', 'ἐμός', 'ἐν', 'ἐπὶ', 'ἐπί', 'ἐὰν', 'ἐάν', 'ἑαυτοῦ', 'ἔτι', 'ἡ', 'ἢ', 'ἤ', 'ὁ', 'ὃδε', 'ὃς', 'ὅδε', 'ὅς', 'ὅστις', 'ὅτι', 'ὑμὸς', 'ὑμός', 'ὑπὲρ', 'ὑπέρ', 'ὑπὸ', 'ὑπό', 'ὡς', 'ὥστε', 'ὦ', 'ξύν', 'ξὺν', 'σύν', 'σὺν', 'τοῖς', 'τᾶς']\n"
```
Comment (clemsciences) [inline 3 Basic NLP.ipynb:1050]: I cannot see the difference between this line and line below.

**Review by clemsciences — CHANGES_REQUESTED** [review]

File: 3 Basic NLP.ipynb (hunk 2 above)
Comment (glaserti) [reply 3 Basic NLP.ipynb:56]: You're absolutely right! The output got messed up bc I didn't edit the nb in venv (I just changed the WordTokenizer parts)

**Pull request closed without merging**